site/_posts/2017-04-17-Announcing-Fine-RC1-Release.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing Fine RC1 Release
+author: chessbyte
+date: 2017-04-17 07:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce Release Candidate 1 for ManageIQ Fine!
+
+Fine is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. In this release, a lot of attention was given to Ansible. However, other providers, REST API, as well as performance also benefit in this release.
+
+* Added **"Ansible Playbook"** to the list of catalog item types.
+* Automate Model: Add Amazon block storage automation models.
+* Adding new report and widgets for Containers and Add option for container performance reports.
+* Add Alert Definitions (MiqAlert) bulk edits support.
+* Middleware: Hawkular Allow adding datawarehouse provider with a port other than 80.
```

Comment:
This list should include the deltas between the beta and RC1.

The rest LGTM! :+1: